wordpress-mobile/WordPress-iOS
# [Stats Refresh] Misc enhancements/changes

Fixes #n/a

cc @SylvesterWilmott 

This includes several enhancements/changes:
- Removing the data bar from Insights Publicize.
- Removing the data bar from Insights Comments (both tabs).
- Abbreviate numbers after 9,999 (instead of 999).
- Updated design for Insights Most Popular Time.
- Adding table footers to both Insights and Period.
- Update the Insights card order.

To test:

---
Data bars:
- Go to Insights.
- Verify Publicize and Comments (both tabs) no longer have data bars.
<img width="250" alt="comments" src="https://user-images.githubusercontent.com/1816888/52741238-1ecd1c00-2f92-11e9-8a96-8ce59341518a.png">
<img width="250" alt="publicize" src="https://user-images.githubusercontent.com/1816888/52741253-27bded80-2f92-11e9-8323-b384bf4e6c16.png">

---
Number abbreviation:
- For all abbreviated values, verify the abbreviate now starts after 9,999. Ex:
<img width="250" alt="abbrev" src="https://user-images.githubusercontent.com/1816888/52742306-b03d8d80-2f94-11e9-8cf3-1e8ecb4686a2.png">

---
Most Popular Time:
- Go to Insights > Most Popular Time.
- Verify it now looks like this:
<img width="250" alt="day_hour_new" src="https://user-images.githubusercontent.com/1816888/52741561-db26e200-2f92-11e9-8659-4ec70c759061.png">

---
Table footers:
- On both Insights and Period, there is now a 20pt row at the bottom.
<img width="250" alt="footer" src="https://user-images.githubusercontent.com/1816888/52741687-31942080-2f93-11e9-9cfe-9a821879df48.png">

---
Insights card order:
- With a _fresh install_ (to clear User Defaults), go to Insights.
- Verify the order is:
  - Latest Post Summary
  - Today's Stats
  - Annual Site Stats
  - All Time Stats
  - Most Popular Time
  - Posting Activity
  - Comments
  - Tags and Categories
  - Follower Totals
  - Followers
  - Publicize
